.github/workflows/system-tests.yml

```diff
@@ -0,0 +1,51 @@
+name: System Tests
+
+on:
+  pull_request:
+    branches:
+      - "**"
+  workflow_dispatch: {}
+  schedule:
+    - cron:  '00 04 * * 2-6'
+
+jobs:
+  system-tests:
+    if: ${{ github.event.pull_request.head.repo.full_name == 'DataDog/dd-trace-py' }}
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+          - weblog-variant: flask-poc
+          - weblog-variant: uwsgi-poc
```

Comment:
In the future, is this the only thing we'll need to tweak when we add more tests?

Reply:
More precisely, it's the same test scenario, but on different target app.

Speaking about the devil, @vdeturckheim just added django, I'll add it.